test/testcafe/spec/EnrollFactorPassword_spec.js

```diff
@@ -1,20 +1,20 @@
 import { RequestMock } from 'testcafe';
-import FactorEnrollPasswordPageObject from '../framework/page-objects/FactorEnrollPasswordPageObject';
+import EnrollPasswordPageObject from '../framework/page-objects/EnrollPasswordPageObject';
 import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
-import xhrFactorEnrollPassword from '../../../playground/mocks/data/idp/idx/factor-enroll-password';
+import xhrAuthenticatorEnrollPassword from '../../../playground/mocks/data/idp/idx/authenticator-enroll-password';
```

Comment:
@haishengwu-okta is this change ok? that means we're moving away from `factor-enroll-password` , or I can create another test file for `authenticator-enroll-password`